content/en/apps/guides/tasks/query-task-data.md

```diff
@@ -0,0 +1,116 @@
+---
+title: "Querying Task Documents"
+linkTitle: Querying Task Documents 
+description: >
+  Querying the data which results from an example task. Notes on the performance implications of tasks.
+relatedContent: >
+  apps/tutorials/tasks-1
+  core/overview/db-schema#tasks
+  core/overview/data-flows-for-analytics
+---
+
+{{% pageinfo %}}
+This guide explains the data which results from tasks and how to query it.
+
+- Write a PostgreSQL query to examine task data
+- Build deeper understanding of task data
+- Present some data considerations of which task authors should remain mindful
+{{% /pageinfo %}}
+
+## Prerequisites
+
+* [Data Flows for Analytics]({{< ref "core/overview/data-flows-for-analytics" >}})
+
+## Querying task data
+The task system running on each user's device is powered by [task documents]({{ < ref "core/overview/db-schema#tasks" >}}) and those task documents sync to the server and to postgres just like a contact or a report. For more information, check out [data flows for analytics]({{< ref "core/overview/data-flows-for-analytics" >}}). Having task documents in Postgres allows system administrators to analyse how users are interacting with tasks.
```

Comment:
```suggestion
The task system running on each user's device is powered by [task documents]({{< ref "core/overview/db-schema#tasks" >}}) and those task documents sync to the server and to PostgreSQL just like a contact or a report. Having task documents in PostgreSQL allows system administrators to analyse how users are interacting with tasks.

{{< see-also page="core/overview/data-flows-for-analytics" title="Data flows for analytics" >}}
```

Reply:
don't like this cross-reference?

Reply:
The cross-reference is fine. My suggestion here is to emphasize it using the 'see-also' tag. Both are fine by me though. The second part of the suggestion is to use have PostgreSQL in full rather than Postgres.

Reply:
Sorry! Maybe I didn't read the whole thing. Accepting